src/std/main.ab

```diff
@@ -266,3 +266,44 @@ pub fun shell_var_get(name: Text): Text {
 pub fun shell_unset(name: Text): Null {
     $unset {name}$?
 }
+
+pub fun shell_text(message: Text, color1: Num, style: Num, color2: Num): Text {
+    return "\e[{style as Text};{color1 as Text};{color2 as Text}m{message}\e[0m"
+}
+
+pub fun bold_text(message: Text): Text {
+    return "\e[1m{message}\e[0m"
+}
+
+pub fun italic_text(message: Text): Text {
+    return "\e[3m{message}\e[0m"
+}
+
+pub fun underlined_text(message: Text): Text {
+    return "\e[4m{message}\e[0m"
+}
+
```

Comment:
I think it's better to start with `text` so that when people start typing that in their IDE the suggestions will show what other alternatives they have.

Example: `text_bold`